widget/label.go

```diff
@@ -30,7 +31,8 @@ func NewLabel(text string) *Label {
 //
 // Since: 2.0.0
 func NewLabelWithData(data binding.String) *Label {
-	label := NewLabel(data.Get())
+	val, _ := data.Get()
+	label := NewLabel(val)
```

Comment:
I don't think this is needed as listener will be notified of current state when added

Reply:
The label needs a value in it’s constructor, surely the known current value is better than a zero that will be changed immediately?

Reply:
Well given it will be changed immediately the value doesn't really matter, but if the options are `""` (which is essentially free), and `data.Get()` (which could be a disk read, or a network call) then I'd opt for the former.

Reply:
If it is "" it will have to change when the new value comes in. If it is set to the current value we avoid the need to re-render the widget.
Of course mostly it won't have rendered yet so maybe it makes less difference than I thought.

Reply:
Pushed. 
I had to change the tests to add a wait for binding to resolve as we can no longer guarantee the start state.